packages/klevu-ui/src/translations/en.json

```diff
@@ -23,6 +23,7 @@
   },
   "productQueryPopup": {
     "tTextFieldPlaceholder": "Enter your question here...",
+    "tAnswerError": "Uh O. There was a problem generating an answer. Please try again after some time.",
```

Comment:
there is another file fi.json, copy the above text into google translate, translate from english to finnish and paste the translated value in the fi.json file.

Reply:
add translation